Exactmins

victor-perez opened the pull request:
Maybe you also like this feature, in some applications, you are only allowed to set 5, 10, 15 etc, so you are not allowed to set exact minutes.
I've added a new option that allows you to disable exact minutes
`exactmins.` default `true`
I've also updated the documentation for this new feature.

victor-perez committed:
ADDED exactmins

victor-perez committed:
version++

victor-perez committed:
Merge pull request #1 from victor-perez/exactmins

Exactmins

danswater commented on bower.json:
```diff
@@ -1,5 +1,5 @@
 {
```
can you also update the entry point of main property?

``` json
  "main": [
     "./dist/js/materialize.clockpicker.js",
     "./dist/css/materialize.clockpicker.css"
   ],
```

this is also useful when injecting files to index.html e.g when using `gulp-inject`

victor-perez replied:
@danswater  good one, I will change that.

danswater replied:
@victor-perez, Thanks! I am amazed that you're still online right now.

victor-perez committed:
# This is a combination of 2 commits.
# The first commit's message is:

merge

# This is the 2nd commit message:

ADDED option: appendto

victor-perez committed:
merge

victor-perez committed:
ADDED option: appendto

victor-perez committed:
Merge pull request #2 from victor-perez/option_target

Option target

victor-perez committed:
version++

victor-perez committed:
ADDED cancelbtn

victor-perez committed:
Merge pull request #3 from victor-perez/cancelbtn

ADDED cancelbtn

victor-perez committed:
Update bower.json

victor-perez committed:
version++

chingyawhao commented:
Ee...there are some conflicts here...

fabyeah commented:
+1 This would be nice.
